examples/plotting/Map_with_cartopy_projections.py

```diff
@@ -0,0 +1,128 @@
+# Robinson
+import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
```

Comment:
`cartopy` has to be added to `conda/environment-test-*.yml` under `# tests` section.

Reply:
`shapely` as well, I think.